containers/daap-docs/src/var/task/swagger.json

```diff
@@ -244,7 +353,10 @@
         }
       },
       "dataProductMetadataSchema": {
-        "$ref": "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform-environments/main/terraform/environments/data-platform/data-product-metadata-json-schema/v1.0.0/moj_data_product_metadata_spec.json"
+        "$ref": "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform-environments/main/terraform/environments/data-platform/data-product-metadata-json-schema/v1.1.0/moj_data_product_metadata_spec.json"
+      },
+      "dataProductSchemaSchema": {
```

Comment:
is this schema describing the full glue schema, or the stripped down schema we are using for the API?

Reply:
stripped down version